rj_gameplay/rj_gameplay/skill/pivot_kick.py

```diff
@@ -54,16 +55,30 @@ def __init__(
         )
         self.capture = capture.Capture(robot)
 
-    def tick(self, robot: rc.Robot, world_state: rc.WorldState, intent: RobotIntent):
-        if self.kick.is_done(world_state):
-            return self.capture.tick(robot, world_state, intent)
-        elif self.pivot.is_done(world_state):
-            return self.kick.tick(robot, world_state, intent)
-        else:
-            return self.pivot.tick(robot, world_state, intent)
+        self._state = "capture"
+
+    def tick(self, world_state: rc.WorldState) -> RobotIntent:
```

Comment:
Needs a way to change the target point on every tick

Reply:
see above reply